Sources/App/Features/Speakers/Models/Migrations/Speaker+Migration+V1.swift

```diff
@@ -0,0 +1,20 @@
+import Fluent
+
+class SpeakerMigrationV1: AsyncMigration {
+    
+    func prepare(on database: Database) async throws {
+        try await database.schema(Speaker.schema)
+             .id()
+             .field("name", .string, .required)
+             .field("biography", .string, .required)
+             .field("twitter", .string)
+             .field("organisation", .string, .required)
+             .field("profile_image", .string, .sql(.default("avatar.png")))
+             .unique(on: "name")
```

Comment:
Are we sure about unique on name? I know its a future problem, but its perfectly feasible to have people over time with the same name

Reply:
T'was a migration already in place, though now you mention it name could very much be an issue with two speakers of the same name going forward. @jw1540 any ideas on why this was the unique over something like UUID, just in case it was for a specific use case.

Reply:
Yeah there’s no real rationale behind this and I’d be more than happy to remove the constraint

Reply:
Removed constraint 👍